### PR TITLE
Added contact and home component

### PR DESCRIPTION
**Pull request by darsh399**
1. home and contactUs component added and created routes for every components

### CODE REVIEW

**Comment by darsh399** [comment]
@aniro45 @abhie53 please review this PR

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
 import './App.css';
 import Page from './components/PageComponent/Page';
-
+import { BrowserRouter as Router } from 'react-router-dom';
```
Comment (aniro45) [inline src/App.js:3]: don't use alise

File: src/components/Body/Body.js [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
+import { Routes, Route} from 'react-router-dom';
+import Home from '../NavigationComponents/Home';
+import ContactUs from '../NavigationComponents/ContactUs';
 
 const Body = () => {
     return (
-        <div>
-            <h1>This is body</h1>
-        </div>
+
```
Comment (aniro45) [inline src/components/Body/Body.js:8]: Format this file

File: src/components/Footer/Footer.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import './Footer.css'
 const Footer = () => {
     return (
         <div>
-
+            <h1>This is Footer</h1>
```
Comment (aniro45) [inline src/components/Footer/Footer.js:6]: Format all files

File: src/components/NavigationComponents/ContactUs.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import './ContactUs.css'
+const ContactUs = () => {
+    return(
+        <div className="main-contact-container">
+        <div className="contact-container">
+            <h1>Contact Us</h1>
+            <div className="contact-info">
+                <div className="conatct-item">
+                <h2>Our Address</h2>
+                <p>123, SVY Biotech, society n. line no.</p>
```
Comment (aniro45) [inline src/components/NavigationComponents/ContactUs.js:11]: use seperate file for messages

**Pull request merged**